gnucash/gnucash-commands.hpp

```diff
@@ -26,13 +26,19 @@
 #define GNUCASH_COMMANDS_HPP
 
 #include <string>
+#include <vector>
 #include <boost/optional.hpp>
 
 using bo_str = boost::optional <std::string>;
+using StrVec = std::vector<std::string>;
 
 namespace Gnucash {
 
+    int quotes_info (void);
     int add_quotes (const bo_str& uri);
+    int report_quotes (const char* source,
```

Comment:
Hmm. I think "add_quotes" and "report_quotes" are better English than "quotes_add" and "quotes_report", but "info_quotes" doesn't really work because "info" doesn't have a verb form. We could call it "check_finance_quotes", which would be more descriptive of what it really does.

Reply:
That's fine with me as well.